learningobjects/tests/test_api.py

```diff
@@ -0,0 +1,62 @@
+"""
+Test api functions.
+"""
+
+from django.test.testcases import TestCase
+
+from django.contrib.auth.models import User
+
+from learningobjects.models import Course
+from learningobjects.api import create_course
+
+# pylint: disable=no-member
```

Comment:
I like the idea of disabling them globally. If we have `thing.whatever()` in our code, and `thing` doesn't have method `whatever`, I'm pretty sure we'll find out pretty quickly.

I think it has to do with metaclass programming. If you parse the Python they actually _don't_ have those methods, but at runtime they do. I know that's how Django models work; I'm just guessing on lxml.